### PR TITLE
FEATURE: Add an anchor to the runtime form to scroll to the right place after submit

### PR DESCRIPTION
**Pull request by mficzel**
Resolves: #14

### CODE REVIEW

File: NodeTypes/Form.fusion [hunk 1]
```diff
@@ -8,21 +8,27 @@ prototype(Sitegeist.PaperTiger:Form) < prototype(Neos.Neos:ContentComponent) {
...
-            renderer = Neos.Fusion.Form:Runtime.RuntimeForm {
-                @cache.context.formClass = '_formClass'
-                namespace = ${q(node).property('_identifier')}
-                attributes.class = ${_formClass}
-                process = Neos.Fusion.Form:Runtime.SingleStepProcess {
-                    content = Sitegeist.PaperTiger:FieldCollection.Renderer {
-                        nodePath = 'fields'
+
+            renderer = Neos.Fusion:Join {
+                anchor = afx`<a id={'form_' + node.identifier}></a>`
```
Comment (jonnitto) [inline NodeTypes/Form.fusion:13]: Shouldn't it be enough to give the form tag the id? I don't like empty anchor that much

Comment (mficzel) [reply NodeTypes/Form.fusion:13]: Not after finishing as then no form is rendered

Comment (jonnitto) [reply NodeTypes/Form.fusion:13]: Hm, ok. But in that case, it should be a prop, so that it is possible to change it (class, aria-labels, tag name, etc)

Comment (jonnitto) [reply NodeTypes/Form.fusion:13]: #23

**Pull request merged**